### PR TITLE
Add split batches

### PR DESCRIPTION
**Pull request by hinthornw**

### CODE REVIEW

File: python/langsmith/client.py [hunk 1]
```diff
@@ -215,6 +216,24 @@ def _serialize_json(obj: Any, depth: int = 0) -> Any:
...
+        The object to serialize.
+    default : Callable[[Any], Any] or None, default=None
+        The default function to use for serialization.
+
+    Returns:
+    -------
+    str
+        The JSON formatted string.
+    """
+    return orjson.dumps(obj, default=_serialize_json)
```
Comment (nfcampos) [inline python/langsmith/client.py:234]: lets add some of the flags like serialize numpy etc etc

Comment (hinthornw) [reply python/langsmith/client.py:234]: is that applied before or after our default serialize json?

Comment (nfcampos) [reply python/langsmith/client.py:234]: before I think

Comment (hinthornw) [reply python/langsmith/client.py]: ah ya otherwise the passthrough options wouldn't make sense. thanks

**Pull request merged**